copy error

### error
```
panic: unsupported type: ptr

goroutine 1 [running]:
panic(0xabb240, 0xc0822c6300)
        D:/Applications/go/src/runtime/panic.go:481 +0x3f4
github.com/mitchellh/reflectwalk.walk(0xd33b20, 0xc08220e380, 0x16, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:129 +0x3e1
github.com/mitchellh/reflectwalk.walkStruct(0xc33780, 0xc08212d440, 0x199, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:264 +0x4bf
github.com/mitchellh/reflectwalk.walk(0xc33780, 0xc08212d440, 0x199, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:126 +0x4d7
github.com/mitchellh/reflectwalk.walkStruct(0xc598e0, 0xc08212d440, 0x199, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:264 +0x4bf
github.com/mitchellh/reflectwalk.walk(0xc598e0, 0xc08212d440, 0x199, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:126 +0x4d7
github.com/mitchellh/reflectwalk.Walk(0xd26d00, 0xc08212d440, 0xd07740, 0xc08220e3f0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/reflectwalk/reflectwalk.go:70 +0x153
github.com/mitchellh/copystructure.Copy(0xd26d00, 0xc08212d440, 0x0, 0x0, 0x0, 0x0)
        E:/Develop/golocal/src/github.com/mitchellh/copystructure/copystructure.go:12 +0x9d
main.main()
```

### I was changed reflectwalk.go line 129
```
case reflect.Bool, reflect.Chan, reflect.Func, reflect.Int, reflect.String, reflect.Invalid, reflect.Ptr:
```
add reflect.Ptr,run success.

--- a/copystructure_test.go
+++ b/copystructure_test.go
@@ -547,3 +547,45 @@ func TestCopy_valueWithLockPointer(t *testing.T) {
 		t.Fatalf("expected errPointerRequired, got: %v", err)
 	}
 }
+
+func TestCopy_mapWithPointers(t *testing.T) {
+	type T struct {
+		S string
+	}
+	v := map[string]interface{}{
+		"a": &T{S: "hello"},
+	}
+
+	result, err := Copy(v)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !reflect.DeepEqual(v, result) {
+		t.Fatal(result)
+	}
+}
+
+func TestCopy_structWithMapWithPointers(t *testing.T) {
+	type T struct {
+		S string
+		M map[string]interface{}
+	}
+	v := &T{
+		S: "a",
+		M: map[string]interface{}{
+			"b": &T{
+				S: "b",
+			},
+		},
+	}
+
+	result, err := Copy(v)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !reflect.DeepEqual(v, result) {
+		t.Fatal(result)
+	}
+}

